src/main/java/book/store/onlinebookstore/dto/BookDto.java

```diff
@@ -0,0 +1,15 @@
+package book.store.onlinebookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {

```

Comment:
since we're using Java 17 - let's use records for DTO :)

Reply:
can't use it due to checkstyle exception